libs/core/src/scss/interaction-state/_hover.scss

```diff
@@ -0,0 +1,54 @@
+@use 'sass:color';
+@use 'sass:meta';
+@use '../utils';
+
+// TODO: Accessibility - Loudness must respect user preferences
+// See https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast
+// TODO: Await UX decision on default loudness and update
+$default-loudness: utils.get-loudness('loud');
+
+@mixin apply-hover($background-color, $color, $property, $loudness: null) {
+  @if meta.type-of($background-color) != color {
+    @error "$background-color must be a valid CSS color value. See https://developer.mozilla.org/en-US/docs/Web/CSS/color_value";
+  }
+
+  @if meta.type-of($color) != color {
+    @error "$color must be a valid CSS color value. See https://developer.mozilla.org/en-US/docs/Web/CSS/color_value";
+  }
```

Comment:
Hmmm, I don't see a DRY'er way to do it. It's two different conditions and two different error messages and they are not dependant on each other. I think it would require more control statements to merge them.

```
if A is not a color
   output "Error A"

if B is not a color
   output "Error B"
```

would become something like

```
if either A or B is not a color

    if it is A that is not a color
        output "Error A"

    else 

    if it is B that is not a color
        output "Error B"
```

which doesn't seem DRY 😁

Do you have any suggestions?